pages.es/common/krunvm.md

```diff
@@ -5,20 +5,20 @@
 
 - Crea una Micro Maquina Virtual basada en Fedora:
 
-`krunvm create {{docker.io/fedora}} --cpus {{number_of_vcpus}} --mem {{memory_in_mbs}} --name "{{name}}"`
+`krunvm create {{docker.io/fedora}} --cpus {{numero_de_vcpus}} --mem {{memoria_en_megabytes}} --nombre "{{nombre}}"`
```

Comment:
```suggestion
`krunvm create {{docker.io/fedora}} --cpus {{numero_de_vcpus}} --mem {{memoria_en_megabytes}} --name "{{nombre}}"`
```

Mandatory fix (multiple violations in the current page): command syntax is broken: `--nombre` option is unknown to `krunvm`.